requirements-tests.txt

```diff
@@ -8,6 +13,8 @@ isort==5.12.0; python_version >= "3.8"            # must match .pre-commit-confi
 mypy==1.3.0
 packaging==23.1
 pathspec>=0.11.1
+
+# Git support.
 pre-commit-hooks==4.4.0                           # must match .pre-commit-config.yaml
 pycln==2.1.5                                      # must match .pre-commit-config.yaml
 pytype==2023.6.2; platform_system != "Windows" and python_version < "3.11"
```

Comment:
Small comment: Given pycln is a formatter and pytype a type-checker, I would've expected to see them in the above section with black and mypy.

Reply:
oh lol, I see.

Might be worth requesting "sort only within groups" as a feature. It feels like a common use-case (I myself am making groups in my requirements file and not using the fixer in my pre-commit for that exact reason).

Removing it atm seems reasonable.